inc/dashboard/namespace.php

```diff
@@ -1169,7 +1169,7 @@ function get_top_data( $start, $end, ?Filter $filter = null ) {
 				'name' => get_the_author_meta( 'display_name', $post->post_author ),
 				'avatar' => get_avatar_url( $post->post_author ),
 			],
-			'views' => $processed[ $post->ID ]['total'],
+			'views' => $processed[ $post->ID ]['total'] ?? 0,
```

Comment:
Is this an unrelated just-in-passing fix?

Reply:
Forgot to do `git commit -p` 🤦‍♂️  - It is yeah. May as well leave it here now.